EvalData/models/base_models.py

```diff
@@ -722,6 +723,20 @@ class TextSegmentWithTwoTargets(TextSegment):
         blank=True, null=True, verbose_name=_('Target context (2)')
     )
 
+    # user for AI-assisted annotation
+    mqm1 = models.TextField(
+        blank=True,
+        verbose_name=_('MQM Annotations'),
```

Comment:
Let's differentiate both fields similar to the ones above:
```suggestion
        verbose_name=_('MQM Annotations (1)'),
```

Reply:
Changed.